includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,21 +1,185 @@
 <?php
 
 class DesignSystemApiController extends WikiaApiController {
+	private $wikiId;
+	private $lang;
+
+	private $data = [
+		'brand-logo' => [
+			'type' => 'brand-logo',
+			'href' => '#',
+			'elements' => [
+				[
+					'type' => 'line-image',
+					'image' => 'company/logo-fandom',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-navigation-logo-fandom',
+					]
+				],
+				[
+					'type' => 'line-image',
+					'image' => 'company/logo-wikia',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-navigation-logo-wikia',
+					]
+				],
+				[
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-header-title',
+				],
+			],
+		],
+		'brand-links' => [
+			[
+				'type' => 'link-branded',
+				'brand' => 'tv',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-brandlink-vertical-tv'
+				],
+				'href' => 'http:\/\/tv.wikia.com'
+			],
+			[
+				'type' => 'link-branded',
+				'brand' => 'games',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-brandlink-vertical-games'
+				],
+				'href' => 'http:\/\/games.wikia.com'
+			],
+			[
+				'type' => 'link-branded',
+				'brand' => 'movies',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-brandlink-vertical-movies'
+				],
+				'href' => 'http:\/\/movies.wikia.com'
+			],
+			[
+				'type' => 'links-branded',
+				'brand' => 'wikis',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-header-title',
+				],
+				'links' => [ // not branded
+					[
+						'type' => 'link-branded',
+						'brand' => 'wikis',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-brandlink-vertical-wikis'
+						],
+						'href' => '#'
+					],
+					[
+						'type' => 'link-branded',
+						'brand' => 'communitycentral',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-brandlink-vertical-communitycentral'
```

Comment:
Can we keep all structure-related comments in one place? : https://docs.google.com/document/d/1oz4IWS_NB11AnL0GS09dWpw1sppCiolhpRR-DVFImwk/edit#